themes/clean/clean.theme.bash

```diff
@@ -18,7 +18,7 @@ function prompt_command() {
 		no_color=${white?}
 	fi
 
-	PS1="${no_color}\u${reset_color}:${blue?}\W/${reset_color} $(scm_prompt_info)${normal?}$ "
+	PS1="${no_color}\u${reset_color?}:${blue?}\W/${reset_color?} $(scm_prompt_info)${normal?}$ "
```

Comment:
This is inconsistent with the `zitron` change 6a19aa537634c08044829090867ce408a559f9f6.